packages/react-components/react-timepicker-compat-preview/src/components/TimePicker/useTimePicker.tsx

```diff
@@ -29,6 +29,7 @@ export const useTimePicker_unstable = (props: TimePickerProps, ref: React.Ref<HT
     endHour = 24,
     hour12 = false,
     increment = 30,
+    formatDateToTimeString,
```

Comment:
```suggestion
    formatDateToTimeString = defaultFormatDateToTimeString,
```

should avoid the ternary

Reply:
They have different type:
 - from user prop `formatDateToTimeString?: (date: Date) => string;`
 - default `formatDateToTimeString(date: Date, options: TimeFormatOptions = {}): string`

I think since TimeFormatOptions are all user props, `formatDateToTimeString` can take just date argument.

Reply:
the default one could just have default options, so that the options argument is optional

Reply:
chat about this on a call - conclusion: I can't do `const { formatDateToTimeString = defaultFormatDateToTimeString } = props` because default formatDateToTimeString needs `options: TimeFormatOptions`.